packages/api/src/praise/transformers.ts

```diff
@@ -99,8 +100,9 @@ const praiseDocumentToDto = async (
     quantifications: await quantificationListTransformer(quantifications),
     giver: userAccountTransformer(giver),
     receiver: userAccountTransformer(receiver),
-    createdAt: createdAt.toISOString(),
-    updatedAt: updatedAt.toISOString(),
+    forwarder: forwarder ? userAccountTransformer(forwarder) : undefined,
+    createdAt: createdAt ? createdAt.toISOString() : '',
+    updatedAt: updatedAt ? updatedAt.toISOString() : '',
```

Comment:
all mongoose models will always have createdAt & updatedAt defined -- so these conditionals are not necessary

Reply:
Oh, yeah, I was debugging an error message and forgot to remove that.